vite.config.ts

```diff
@@ -15,6 +15,9 @@ export default defineConfig(({ mode }) => {
   const outDir = 'dist'
 
   const PISA_URL = env.PISA_URL
+  const enablePisa = env.ENABLE_PISA_ONLY_ON_PISA_BRANCH
+    ? env.CF_PAGES_BRANCH === 'pisa' && !!PISA_URL
+    : !!PISA_URL
```

Comment:
* [ ] FYI the `enablePisa` logic could be moved to the server setup, e.g., to the "build command" option. This way, we'd be able to hide this very specific condition from the app config.

See https://stackoverflow.com/a/2440950

Reply:
Not sure how this would look like.

Reply:
I went with:
```bash
PISA_URL=$([ "$CF_PAGES_BRANCH" = "pisa" ] && echo $PISA_URL) npm run build
```

Reply:
FYI Another option that came to my mind is a 2nd Pages setup building only the main branch with a different env. This one-time setup would save us from permanently having to duplicate the main branch.

Reply:
As far as I know Cloudflare Pages does not allow to create two different applications, that link to the same git repository.

Reply:
I see. Very unfortunate.